src/PlantsIdentifierAPI/Dtos/PlantDto.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace PlantsIdentifierAPI.Dtos
+{
+    public class PlantDto
+    {
+        public Guid ID { get; set; }
+        public string PhotoURL { get; set; }
+        public string CommonName { get; set; }
+
+        public override string ToString()
+        {
+            return $"PlantDto(ID={ID},CommonName={CommonName},PhotoURL={PhotoURL})";
```

Comment:
Hey, do we need this? Or does is this mandatory for automapper?

Reply:
No, it isn't mandatory. I usually override the `ToString` method because I find it useful during the debug (the debugger shows this string when you evaluate the value of an object). If you prefer, I can remove it.